include/SFML/Graphics/Color.inl

```diff
@@ -22,31 +22,23 @@
 //
 ////////////////////////////////////////////////////////////
 
-////////////////////////////////////////////////////////////
-// Headers
-////////////////////////////////////////////////////////////
-#include <SFML/Graphics/Color.hpp>
-#include <algorithm>
-
 
-namespace sf
-{
 ////////////////////////////////////////////////////////////
 // Static member data
 ////////////////////////////////////////////////////////////
-const Color Color::Black(0, 0, 0);
-const Color Color::White(255, 255, 255);
-const Color Color::Red(255, 0, 0);
-const Color Color::Green(0, 255, 0);
-const Color Color::Blue(0, 0, 255);
-const Color Color::Yellow(255, 255, 0);
-const Color Color::Magenta(255, 0, 255);
-const Color Color::Cyan(0, 255, 255);
-const Color Color::Transparent(0, 0, 0, 0);
+inline const Color Color::Black{0, 0, 0};          //!< Black predefined color
+inline const Color Color::White{255, 255, 255};    //!< White predefined color
+inline const Color Color::Red{255, 0, 0};          //!< Red predefined color
+inline const Color Color::Green{0, 255, 0};        //!< Green predefined color
+inline const Color Color::Blue{0, 0, 255};         //!< Blue predefined color
+inline const Color Color::Yellow{255, 255, 0};     //!< Yellow predefined color
+inline const Color Color::Magenta{255, 0, 255};    //!< Magenta predefined color
+inline const Color Color::Cyan{0, 255, 255};       //!< Cyan predefined color
+inline const Color Color::Transparent{0, 0, 0, 0}; //!< Transparent (black) predefined color
```

Comment:
Why not directly `constexpr`?

Also, let's stay with the current SFML code style for now (constructor parentheses), otherwise we get an inconsistent code base. See also section "Uniform initialization" in [this forum thread](https://en.sfml-dev.org/forums/index.php?topic=28340.0).

Reply:
It doesn't work, see https://stackoverflow.com/questions/11928089/static-constexpr-member-of-same-type-as-class-being-defined -- super annoying. I reverted this change anyway because VS2017 doesn't work properly with `inline` variables AFAICS.

Reply:
What we should do here is change `sf::Color::Red` to `sf::Colors::Red` with `Colors` being a namespace or class, then we can define all colors as `constexpr`. But that's for a subsequent PR.

Reply:
I created a StackOverflow question here: https://stackoverflow.com/questions/70356275/working-around-limitation-of-constexpr-static-data-member-with-same-type-as-en

Reply:
Looks like there might be a way with https://stackoverflow.com/a/70356507, do you think that's applicable here?

Reply:
> Looks like there might be a way with https://stackoverflow.com/a/70356507, do you think that's applicable here?

I am considering a few options, I'd like to merge this PR as is and deal with the constants in a separate one.